CSC307Fall2018/team-too
# Add product endpoints

# Summary
Closes #67 

1. Added endpoints such that create/update/deleteProduct can be queried locally
2.  **createProduct mutation still creates a duplicate product**
3. Fixed duplicate and comma-less resolvers
4. Made product.Stock queryable from the ProductInfo.vue

# QA
- Run the server 
- Open the 
- Assess that the queries and mutations work as expected
